types/node/crypto.d.ts

```diff
@@ -3663,7 +3663,7 @@ declare module "crypto" {
          * Third-party code should be prepared to handle both possible entry formats.
          * @since v15.6.0
          */
-        readonly infoAccess: string | undefined;
+        infoAccess: string | undefined;
```

Comment:
This is a readonly getter, no?

https://github.com/nodejs/node/blob/741975041995a272ff7e378bcbb6d6fa4b93f38f/lib/internal/crypto/x509.js#L198-L205

Reply:
Ah yes. My bad.